lms/djangoapps/staticbook/views.py

```diff
@@ -147,7 +146,6 @@ def html_index(request, course_id, book_index, chapter=None):
     course_key = CourseKey.from_string(course_id)
     course = get_course_with_access(request.user, 'load', course_key)
     staff_access = bool(has_access(request.user, 'staff', course))
-    notes_enabled = notes_enabled_for_course(course)
```

Comment:
`notes_enabled_for_course` refers to the Notes API which is still supported, not annotations.

Reply:
Ugh, no, nevermind. That checks `ENABLE_STUDENT_NOTES`, whereas the currently supported Notes API checks `ENABLE_EDXNOTES`. Kill it with fire.